stateful-functions-flink/stateful-functions-flink-core/src/main/java/com/ververica/statefun/flink/core/message/MessageFactoryType.java

```diff
@@ -19,5 +19,6 @@
 public enum MessageFactoryType {
   WITH_KRYO_PAYLOADS,
   WITH_PROTOBUF_PAYLOADS,
-  WITH_RAW_PAYLOADS
+  WITH_RAW_PAYLOADS,
+  WITH_PROTOBUF_MULTILANG
```

Comment:
For the sake of consistent naming, perhaps rename this to `WITH_PROTOUF_MULTILANG_PAYLOADS`?